02-IDE_exercises/solutions/07b-project/reverse_list.py

```diff
@@ -2,13 +2,19 @@
 # Date: 10/29/2020
 # Description: Mutates a given list and by the order of the elements of teh list
 
-def reverse_list(lst):
+def reverse_list(lst: any) -> None:
```

Comment:
I don't believe `any` is a type hint. The recommendation I ready is to call `object` for the `Any` class. This should be type hinted as `list[object]`